src/strategies/Strategy.ts

```diff
@@ -0,0 +1,6 @@
+import { Context } from 'telegraf';
+import { ChatData } from 'Bot';
+
+export default abstract class Strategy {
+    abstract execute(ctx: Context, chatData: ChatData): void;
+}
```

Comment:
I can see a pattern in all strategies, what about to move a strategy `welcome message` to an abstract method
```suggestion
abstract welcomeReply(ctx: Context)
```
We can think about naming